src/modules/core/producer_blank.c

```diff
@@ -0,0 +1,69 @@
+/*
+ * producer_blank.c
+ * Copyright (C) 2023 Meltytech, LLC
+ *
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 2.1 of the License, or (at your option) any later version.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this library; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+ */
+
+#include <framework/mlt_frame.h>
+#include <framework/mlt_producer.h>
+
+#include <stdlib.h>
+
+static int producer_get_frame(mlt_producer self, mlt_frame_ptr frame, int index)
+{
+    // Generate a frame
+    *frame = mlt_frame_init(MLT_PRODUCER_SERVICE(self));
+
+    if (*frame != NULL) {
```

Comment:
No need to compare with NULL  - not consistent with the majority of the code.

Reply:
done